index.html

```diff
@@ -53,5 +53,226 @@ <h3 class="job-title">
+                    sign-ups required. Has been the industry's standard.
+                </p>
+                <ul class="work2Languages">
+                    <li> <button class="work2Button" type="button">HTML </button> </li>
+                    <li> <button class="work2Button" type="button">Bootstrap</button> </li>
+                    <li> <button class="work2Button" type="button">Ruby</button> </li>
+                </ul>
+                <button class="work2Link">See project</button>
+            </div>
+            <div class="work2">
+                <h3 class="work2Title">
+                    Profesional Art Printing Data
+                </h3>
+                <p class="work2Description">
+                    A daily selection of privately personalized reads; no accounts or 
+                    sign-ups required. Has been the industry's standard.
+                </p>
+                <ul class="work2Languages">
+                    <li> <button class="work2Button" type="button">HTML </button> </li>
+                    <li> <button class="work2Button" type="button">Bootstrap</button> </li>
+                    <li> <button class="work2Button" type="button">Ruby</button> </li>
+                </ul>
+                <button class="work2Link">See project</button>
+            </div>
+            <div class="work2">
+                <h3 class="work2Title">
+                    Profesional Art Printing Data
+                </h3>
+                <p class="work2Description">
+                    A daily selection of privately personalized reads; no accounts or 
+                    sign-ups required. Has been the industry's standard.
+                </p>
+                <ul class="work2Languages">
+                    <li> <button class="work2Button" type="button">HTML </button> </li>
+                    <li> <button class="work2Button" type="button">Bootstrap</button> </li>
+                    <li> <button class="work2Button" type="button">Ruby</button> </li>
+                </ul>
+                <button class="work2Link">See project</button>
+            </div>
+        </div>
+    </section>
+    <!-- About Me section -->
+    <section>
+        <div class="aboutMe">
+            <h2 class="aboutMeTitle">
+                About me
+            </h2>
+            <p class="aboutMeDescription">
+                Hello I’m a software developer! I can help you build a product , feature or 
+                website Look through some of my work and experience! If you like what you 
+                see and have a project you need coded, don’t hesitate to contact me.
+            </p>
+            <div class="resumeLinkContainer">
+                <button class="resumeLink">Get my resume</button>
+            </div>
+            <div class="developer"> 
+                <img src="images/aboutMe background.png" alt="developer with his laptop">
+            </div>
```

Comment:
- [ ] kindly consider adding this image as a background image as its a decoration and this will help you to full-fill the below requirement 
- > The section has the proper background image (decorations are always backgrounded images, not `img` tags)